store.html

```diff
@@ -58,7 +58,106 @@
+                      <input
+                        class="name"
+                        id="first name"
+                        type="text"
+                        name="first name"
+                        required
+                      />
+                    </div>
+                    <div class="last_name_wrapper"
+                      <label for="last name">Last name <span class="required">*</span></label>
+                      <input
+                        class="name"
+                        type="text"
+                        id="last name"
+                        name="last name"
+                        required
+                      />
+                    </div>
+                </div>
+
+                <label for="address">Address <span class="required">*</span></label>
+                <input type="text" id="address" name="address" required />
+
+                <label for="address2">Address 2</label>
+                <input type="text" id="address2" name="address2" />
+
+                <div class ="city_wrapper">
+                    <div class="city">
+                        <label for="city">City <span class="required">*</span></label>
+                        <select class ="city_options" name="city" id="city" required>
+                          <option value="select your city">Select your city...</option>
+                          <option value="london">London</option>
+                          <option value="paris">Paris</option>
+                          <option value="rome">Rome</option>
+                        </select>
+                    </div>
+                    <div class="postcode"> 
+                          <label for="postcode">Postcode <span class="required">*</span></label>
+                          <input type="text" id="postcode" name="postcode" required />
+                    </div>
+                </div>
+
+                 <!--  __________________ radio type ___________________________ -->
+
+                <div class = "select_color"> 
+
+                    <p class="radio_title">Select a color</p>
+
+                        <div class="radio_wrapper">
+                              <input class =  "radio"
+                                type="radio"
+                                id="karmaorange"
+                                value="karmaorange"
+                                name="color"
+                              />
+                              <label class="radio_label" for="orange">Karma orange</label>
+                              <input class =  "radio"
+                                type="radio"
+                                id="spacegray"
+                                value="space_gray"
+                                name="color"
+                              />
+                              <label for="gray">Space gray</label>
+                        </div>
```

Comment:
Nit: Some of the spacing here is inconsistent with the rest of the page. It'll make it easier to match the components and read it if the spacing consistent